### ISSUE TITLE
Support for keyboard accessibility

### ISSUE THREAD

**Issue by 9inpachi** [HSF/phoenix#308]
Currently, not everything that is clickable can be accessed through keyboard (by tabbing through elements). Keyboard accessibility can be added by using either the `<button>` element wherever possible or adding `role="button"`, `tabIndex`, `onKeyPress` to non-button elements.

We should have complete keyboard accessibility for UI menu and Phoenix menu.

See for example in React:
https://github.com/9inpachi/react-circular-menu/blob/master/src/CircleButton/CircleButton.tsx
https://github.com/9inpachi/react-circular-menu/blob/master/src/helpers/buttonA11yProps.ts
